bin/livy-server

```diff
@@ -32,9 +54,115 @@ if [ ! -d "$LIBDIR" ]; then
   exit 1
 fi
 
-CLASSPATH="$LIBDIR/*:$LIVY_HOME/conf:$CLASSPATH"
 
-exec java \
-	$LIVY_SERVER_JAVA_OPTS \
-	-cp "$CLASSPATH" \
-	com.cloudera.livy.server.Main
+
+
+if [ "$LIVY_IDENT_STRING" = "" ]; then
+  export LIVY_IDENT_STRING="$USER"
+fi
+
+
+# get log directory
+if [ "$LIVY_LOG_DIR" = "" ]; then
+  export LIVY_LOG_DIR="${LIVY_HOME}/logs"
+fi
+mkdir -p "$LIVY_LOG_DIR"
```

Comment:
Also you should only do any of this stuff if the command is `start`. None of this is needed (or makes much sense) for `stop`.